content/cloud/custom-domain.md

```diff
@@ -29,6 +29,8 @@ Currently, you can apply a maximum of one custom domain to your Spin application
 
 Note that this propagation is generally performed in a reasonable time frame (minutes/hours). However, in some cases, propagation can take up to 72 hours. You can check the DNS record types for your domain name using the [Google Public DNS](https://dns.google/) service.
 
+Be aware that updating your domain's nameserver records will cause any additional records you've set through you registrar to no longer take effect. This includes, for example, TXT or MX records for services like Google Workspace. You can work around this by connecting your Fermyon Cloud app to a subdomain. But if you need to use the apex domain (example.com rather than app.example.com) and you also need to set additional DNS records, you should choose a different hosting service for the time being.
```

Comment:
Thanks for making this edit! I made a few wording adjustments with this inline suggestion. One other consideration to take or leave: would it be possible to move this suggestion towards to the top of the article under a `## Limitations` header so users are aware a bit earlier in the tutorial in case it is a blocker for them? 

```suggestion
>> Additional records for custom domains are not supported on Fermyon Cloud at this time. This means any additional records (such as TXT or MX) you've set through your registrar no longer take effect once you've delegated your domain. If you need to assign your Spin application to an apex domain (`example.com` rather than `app.example.com`) _and_ you also need to set additional DNS records associated with that domain, please share this [feedback here](https://github.com/fermyon/feedback/issues/47). In the meantime, please consider delegating a subdomain or using a Fermyon custom subdomain for your Spin application instead.
```